addons/sms/wizard/sms_composer.py

```diff
@@ -74,14 +75,15 @@ def default_get(self, fields):
         readonly=False, store=True, required=True)
 
     @api.depends('res_ids_count', 'active_domain_count')
-    @api.depends_context('default_composition_mode')
     def _compute_composition_mode(self):
         for composer in self:
-            if self.env.context.get('default_composition_mode') == 'guess' or not composer.composition_mode:
+            if self.env.context.get('composition_mode_default') == 'guess' or not composer.composition_mode:
```

Comment:
Is it safe to remove the `@api.depends_context()`?

Reply:
It's a stored field on a wizard, the field should never have different values in cache.